source/includes/_endpoint-variable.md

```diff
@@ -217,7 +224,7 @@ omitted, one will be generated from the required "name".
 You may also include "values", which will create the column of data
 corresponding to this variable definition. See [Importing Data: Column-by-column](#column-by-column) for details and examples.
 
-You may instead also include an "expr" to derive a variable as a function of
+You may instead also include an "derivation" to derive a variable as a function of
```

Comment:
We accept both, but we changed several months ago to prefer "derivation", we still accept expr as legacy.

We changed this because `derivation` is the attribute exposed on the API, so it made sense to have `derivation` IN and `derivation` OUT. 

https://github.com/Crunch-io/backend-http/blob/master/cr/server/api/controllers/variables/catalog.py#L549